bucket/manjarowsl.json

```diff
@@ -0,0 +1,62 @@
+{
+    "##": "'rootfs' must be a 'real' directory (at least on Windows 10 RS4)",
+    "version": "20211001",
+    "description": "Install Manjaro as a WSL Instance",
+    "homepage": "https://github.com/sileshn/ManjaroWSL",
+    "license": "MIT",
+    "notes": "Even when you are logging in as 'root', some operations (like service command) require Windows administrator privileges",
+    "url": "https://github.com/sileshn/ManjaroWSL/releases/download/20211001/ManjaroWSL.zip",
+    "hash": "c7f08056fcdc4c1a885c9269a7b17d289d295bab742693907e2575279eed6e91",
+    "post_install": [
+        "$installable = $true",
+        "$user = [Security.Principal.WindowsIdentity]::GetCurrent() -as [Security.Principal.WindowsPrincipal]",
+        "$permission = Get-Acl $persist_dir | Select-Object -ExpandProperty Access | Where-Object {",
+        "    ($user.IsInRole($_.IdentityReference)) -and `",
+        "    ($_.FileSystemRights.ToString() -eq 'FullControl') -and `",
+        "    ($_.InheritanceFlags -band 3) -and `",
+        "    ($_.PropagationFlags.ToString() -ne 'NoPropagateInherit')",
+        "}",
+        "if ($null -eq $permission) {",
+        "   warn 'Full Control access to the scoop directory is necessary to install WSL distribution.'",
+        "   warn 'Change directory security and reinstall ManjaroWSL.'",
+        "   $installable = $false",
+        "}",
+        "if ($null -eq (Get-Command 'wslconfig' -ErrorAction SilentlyContinue)) {",
+        "    warn 'WSL appears not to be enabled!'",
+        "    warn 'Run ''Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux'''",
+        "    warn 'from an elevated PowerShell. Restart your computer when prompted and reinstall ManjaroWSL.'",
+        "    $installable = $false",
+        "}",
+        "if ($installable) {",
+        "    $installable = !(Invoke-ExternalCommand \"$dir\\Manjaro.exe\" 'isregd')",
+        "}",
+        "if ($installable) {",
+        "    warn 'DO NOT KILL THE PROCESS. Installation is still running!'",
+        "    Copy-Item \"$dir\\Manjaro.exe\" \"$persist_dir\\data\\Manjaro.exe\"",
+        "    $res = Invoke-ExternalCommand \"$persist_dir\\data\\Manjaro.exe\" 'install', \"$dir\\rootfs.tar.gz\"",
+        "    Remove-Item \"$persist_dir\\data\\Manjaro.exe\" -Force",
+        "    if(!$res) { error 'ManjaroWsl installation failed!'; return }",
+        "}",
+        "Remove-Item \"$dir\\rootfs.tar.gz\" -Force"
```

Comment:
Shouldn't this line be in the above if block too?

Reply:
No, I don't think it should be. There is a check in the first if block that changes `$installable` (also just copied that from #5224 so it should be suggested there too).

But I do think there is a better way to write that.